edx_django_utils/monitoring/internal/code_owner/utils.py

```diff
@@ -103,6 +109,58 @@ def get_code_owner_mappings():
     return _PATH_TO_CODE_OWNER_MAPPINGS
 
 
+def _get_catch_all_code_owner():
+    """
+    If the catch-all module "*" is configured, return the code_owner.
+
+    Returns:
+        (str): code_owner or None if no catch-all configured.
+
+    """
+    try:
+        code_owner = get_code_owner_from_module('*')
+        return code_owner
+    except Exception:  # pylint: disable=broad-except; #pragma: no cover
+        return None
+
+
+def _set_code_owner_attribute_from_module(module):
+    """
+    Updates the code_owner and code_owner_module custom attributes.
+    """
+    set_custom_attribute('code_owner_module', module)
+    code_owner = get_code_owner_from_module(module)
+    if code_owner:
+        set_custom_attribute('code_owner', code_owner)
+        return
+
+    code_owner = _get_catch_all_code_owner()
+    if code_owner:
+        set_custom_attribute('code_owner', code_owner)
```

Comment:
This block can be simplified:

```
code_owner = get_code_owner_from_module(module)
if not code_owner:
    code_owner = _get_catch_all_code_owner()
set_custom_attribute('code_owner', code_owner)
```

Reply:
It would still need a final `if code_owner` check, but makes sense.

Reply:
FYI: I refactored the two places with this code.  It didn't shrink lines of code in one case, but it did remove the duplication around setting the custom attribute.